glide.yaml

```diff
@@ -38,3 +38,4 @@ import:
 - package: golang.org/x/tools
   subpackages:
   - go/gcexportdata
+- package: github.com/Sirupsen/logrus
```

Comment:
logrus a logger implementation that only matters for our internal code. If grafton was useful as a library for others, and not just as the cli tool now, I'd advocate quite strongly for using an interface to set a logger instead of depending on the concrete logrus. As it stands though, it just feels a little strange but is probably ok.